src/main.rs

```diff
@@ -211,3 +167,66 @@ fn main() -> Result<ExitCode, FlakeCheckerError> {
 
     Ok(ExitCode::SUCCESS)
 }
+
+#[cfg(feature = "allowed-refs")]
+#[derive(Parser)]
+struct Cli {
+    // Check to make sure that Flake Checker is aware of the current supported branches.
+    #[arg(long, hide = true)]
+    check_allowed_refs: bool,
+
+    // Check to make sure that Flake Checker is aware of the current supported branches.
+    #[arg(long, hide = true)]
+    get_allowed_refs: bool,
+}
+
+#[cfg(feature = "allowed-refs")]
+fn main() -> Result<ExitCode, FlakeCheckerError> {
+    let Cli {
+        check_allowed_refs,
+        get_allowed_refs,
+    } = Cli::parse();
+
+    if !get_allowed_refs && !check_allowed_refs {
+        panic!("You must select either --get-allowed-refs or --check-allowed-refs");
+    }
```

Comment:
Clap has the ability to set required arguments. Probably better to use that than panic.

Reply:
In this case, you're supposed to set one or the other

Reply:
And I think it's okay to panic here because, as mentioned above, it's only used in CI